aflr/helper_classes.py

```diff
@@ -0,0 +1,70 @@
+from .api_request import APIRequest
+
+
+class ListableResource(APIRequest):
+    def __init__(self):
+        super().__init__()
+
+    @classmethod
+    def list(cls, **args):
+        if args:
+            return cls._get_request(request_params=args)
+        else:
+            return cls._get_request()
+
+
+class CreatableResource(APIRequest):
+    def __init__(self):
```

Comment:
is this init actually doing something?

Reply:
removed all init methods for helper classes